src/main/steps/enter-your-address/content.ts

```diff
@@ -0,0 +1,68 @@
+import { Checkbox } from '../../app/case/case';
+import { TranslationFn } from '../../app/controller/GetController';
+import { FormContent } from '../../app/form/Form';
+import { isInvalidPostcode } from '../../app/form/validation';
+
+const en = {
+  title: 'Enter your postal address',
+  postcode: 'Enter a UK postcode',
+  findAddress: 'Find address',
+  notUK: 'I cannot enter a UK postcode',
+  errors: {
+    yourAddressPostcode: {
+      required: 'You have not entered your address. Enter your address before continuing.',
+      invalid: 'You have not entered a valid UK postcode. Enter a valid UK postcode before continuing.',
+    },
+  },
+};
+
+// @TODO translations
+const cy = en;
+
+export const form: FormContent = {
+  fields: {
+    yourAddressPostcode: {
+      type: 'text',
+      classes: 'govuk-label govuk-input--width-10',
+      label: l => l.postcode,
+      labelSize: null,
+      hideError: true,
+      validator: (value, form) => {
+        if (form?.myAddressIsInternational === Checkbox.Checked) {
```

Comment:
you've used the form variable name twice here

Reply:
also when is it not set? Are you just slapping ? on everything? :D

Reply:
👍 Good spot, I've renamed this and updated the types and other calls so I don't need to optionally chain this